specs/Standards/NonFungibleToken/ApprovalManagement.md

```diff
@@ -2,7 +2,7 @@
 
 ## [NEP-178](https://github.com/near/NEPs/discussions/178)
 
-Version `1.0.0`
+Version `1.0.1`
```

Comment:
What semver rules are we using? I'm for this change but a patch version for a breaking change is a bit odd

Reply:
Yea I could change this to 2.0.0 - i'm not sure what the convention is.

Reply:
I'll go with this - I couldn't find anything either.